features/support/paths.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+# rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
In general, when you disable rubocop stuff, remember to enable it again at the end of the class/method/block/file.
That being said, I do not think that we should merge this change.

Reply:
Yes I know... but I think it's silly rubocop don't have a page level disabling feature 😩